apps/freeablo/fagui/guimanager.cpp

```diff
@@ -430,8 +430,8 @@ namespace FAGui
             nk_button_label_styled(ctx, &dummyStyle, "");
             if (nk_widget_is_mouse_click_down_inactive(ctx, NK_BUTTON_LEFT))
             {
-                inv.inventoryMouseLeftButtonDown(Misc::Point{int32_t((ctx->input.mouse.pos.x - invTopLeft.x - ctx->current->bounds.x) / cellSize),
```

Comment:
Does this really help though? We're still pasing exact cell not based on item size. But for certain items its center is in the middle of the cell and for others it's in corner of one cell or in the middle of cell side, so it should condier size of the item before rounding.

Reply:
Hmm yeah, this does break it for other items. I'll do a "proper this time I swear" fix later :p